lib/redis/store/interface.rb

```diff
@@ -6,7 +6,7 @@ def get(key, options = nil)
       end
 
       def set(key, value, options = nil)
-        super(key, value)
+        super(key, value, options || {})
```

Comment:
I tried that first, but since other classes also use options = nil and then call set(key, value, options), options would still be nil (even if the definition is def set(key, value, options = {})). So, I would have to change quite a few files in order to achieve the same effect. But, if changing all set methods to options={} does not have any side effects, I can change them, if you prefer.